source/documentation/other-topics/statefulsets.md

```diff
@@ -0,0 +1,122 @@
+### StatefulSets (Pods with Persistent Volumes)
+
+#### Introduction
+
+Like a Deployment, a [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) manages Pods that are based on an identical container spec. Unlike a Deployment, a StatefulSet maintains a sticky identity for each of their Pods. These pods are created from the same spec, but are not interchangeable: each has a persistent identifier (UID) that it maintains across any rescheduling. It also provides guarantees ordering and uniqueness of the Pods.
```

Comment:
Perhaps:
"It also provides guarantees _on the_ ordering and [...]"

Reply:
Corrected